### ISSUE TITLE
Debug-build on Windows fails unless python27_d.lib is present

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF#42]
It seems that Python.h automatically instructs the linker to use python27_d.lib (we don't!). On my system (Anaconda) this results in a linking error as this library is not distributed.

This appears to be a known problem, see e.g. https://stackoverflow.com/questions/16200997/why-doesnt-include-python-h-work 

As we cannot control the Swig output, one (rather horrible) trick is to make your own Python.h as in
https://github.com/malaterre/GDCM/blob/master/Wrapping/Python/Python.h.in

**Comment by casperdcl** [comment]
Yes, I've used said horrible trick before. We could possibly alternatively get away with creating a temporary symlink `python27_d.lib -> python27.lib`

**Comment by KrisThielemans** [comment]
reading more on this, these tricks seems unsafe (as the debug build of python isn't compatible with the normal one). So we won't fix this.
